tailwind.config.js

```diff
@@ -1,12 +1,15 @@
 module.exports = {
-  content: ["./pages/**/*.{js,ts,jsx,tsx}", "./lib/**/*.{js,ts,jsx,tsx}"],
+  content: ['./pages/**/*.{js,ts,jsx,tsx}', './lib/**/*.{js,ts,jsx,tsx}'],
   variants: {
     extend: {
-      margin: ["last"],
+      margin: ['last'],
     },
   },
   theme: {
     extend: {},
+    screens: {
+      lg: '896px',
```

Comment:
@rlogank before this change `lg` matched `max-w-5xl` so all was good because the style shifted when the screen reached `max-w-5xl`. But then when I changed the max width to `max-w-4xl` the style remained mobile-optimized even when the viewport was wider than `max-w-4xl`. So here I reduced the width of the `lg` breakpoint to match `max-w-4xl` and now things look good again.

My concern is that `lg` is a Tailwind standard and overriding it might cause confusion to someone working on the code. Do you think we should rename it to something else and use that new name in the code?